### PR TITLE
fix: remove labels on releases when successful

### PR DESCRIPTION
**Pull request by crwilcox**

### CODE REVIEW

File: .github/workflows/python-publish.yml [hunk 1]
```diff
@@ -151,4 +151,31 @@ jobs:
...
+      - uses: actions/github-script@v3
+        id: untag-release
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const [owner, repo] = process.env.GITHUB_REPOSITORY.split('/');
+            await github.issues.addLabels({
+              owner,
+              repo,
+              issue_number: ${{steps.tag-release.outputs.pr}},
```
Comment (crwilcox) [inline .github/workflows/python-publish.yml:172]: as this job doesn't stem from the release pr, we don't know what PR it was that cause this release to publish. @bcoe ?

**Comment by parthea** [comment]
@bcoe  Please can you take a look?

**Comment by bcoe** [comment]
@crwilcox @parthea [releasetool](https://github.com/googleapis/releasetool) is able to update and remove labels, see [this release for example](https://github.com/googleapis/nodejs-secret-manager/pull/2800).

I wonder if it's perhaps just not configured properly for this repository, or if there was a temporary API issue?

As you say @crwilcox, you lack context in the action to remove labels, since the release is actually handled by the GitHub app on this repository I believe?

**Comment by bcoe** [comment]
_Edit: aha, just saw the publish.yml job._

Could we be using releasetool for this repo to perform the publish?

**Comment by parthea** [comment]
Hey @crwilcox,

I'm going to close this PR due to inactivity but please feel free to re-open it.

**Pull request closed without merging**